### ISSUE TITLE
Crash with v2.9.1

### ISSUE THREAD

**Issue by yukitoto** [mixpanel/mixpanel-swift#109]
When I try to put 4 fingers on my device to connect the device for AB test, Mixpanel always crash.
This is crash log. The version of this library is v2.1.9.
```
Crashed: com.apple.main-thread
0  Mixpanel                       0x101112c6c UIView.mp_text() -> String? (UIViewSelectors.swift:155)
1  Mixpanel                       0x101112cac @objc UIView.mp_viewId() -> String? + 48
2  Mixpanel                       0x1011113cc @objc UIView.mp_varA() -> NSString? + 40
3  Foundation                     0x1869845f0 -[NSObject(NSKeyValueCoding) valueForKey:] + 264
4  Mixpanel                       0x101022df8 ObjectSerializer.getPropertyValue(of : inout AnyObject, propertyDescription : PropertyDescription, context : ObjectSerializerContext) -> Any (ObjectSerializer.swift:134)
5  Mixpanel                       0x101029054 specialized ObjectSerializer.visitObject(AnyObject?, context : ObjectSerializerContext) -> () (ObjectSerializer.swift)
6  Mixpanel                       0x101029e04 specialized ObjectSerializer.getSerializedObjects(rootObject : AnyObject) -> [String : AnyObject] (ObjectSerializer.swift)
7  Mixpanel                       0x10105849c specialized SnapshotRequest.(responseCommand(connection : WebSocketWrapper) -> Operation?).(closure #1).(closure #2) (ObjectSerializer.swift)
8  Mixpanel                       0x101058354 partial apply for SnapshotRequest.(responseCommand(connection : WebSocketWrapper) -> Operation?).(closure #1).(closure #2) (SnapshotMessage.swift)
9  libdispatch.dylib              0x184e469a0 _dispatch_client_callout + 16
10 libdispatch.dylib              0x184e577a8 _dispatch_barrier_sync_f_slow_invoke + 304
11 libdispatch.dylib              0x184e469a0 _dispatch_client_callout + 16
12 libdispatch.dylib              0x184e4b5e8 _dispatch_main_queue_callback_4CF + 996
13 CoreFoundation                 0x185f3d0c0 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 12
14 CoreFoundation                 0x185f3acdc __CFRunLoopRun + 1572
15 CoreFoundation                 0x185e6ad94 CFRunLoopRunSpecific + 424
16 GraphicsServices               0x1878d4074 GSEventRunModal + 100
17 UIKit                          0x18c123130 UIApplicationMain + 208
18 sample                         0x1000a581c main (AppDelegate.swift:25)
19 libdyld.dylib                  0x184e7959c start + 4
```

**Comment by yarneo** [comment]
Thanks @yukitoto I just pushed to master a fix for this. Sorry for the inconvenience! Will be happy to send out a new release start of next week, or feel free to use the master branch for now.

**Comment by yarneo** [comment]
fixed in v2.2.0 ( https://github.com/mixpanel/mixpanel-swift/releases/tag/v2.2.0 ) !
